docs/chapter-3/lesson-3.3.md

```diff
@@ -135,7 +135,9 @@ For `n` = 9, this will give the required output:
 1,2,3,4,5,6,7,8,9
 ```
 
-Whenever we use the `#!py print()` function, it prints the expression passed to it and immediately follows it up by printing a newline. This is the default behaviour of `#!py print()`. It can be altered by using a special argument called `end`. The default value of `end` is set to the newline character. So, whenever the end argument is not explicitly specified in the print function, a newline is appended to the input expression by default. In the code given above, by setting `end` to be a comma, we are forcing the `#!py print()` function to insert a comma instead of a newline at the end of the expression passed to it. It is called `end` because it is added at the end. To get a better picture, consider the following code:
+Whenever we use the `#!py print()` function, it prints the expression passed to it and immediately follows it up by printing a newline. This is the default behaviour of `#!py print()`. It can be altered by using a special argument called `end`. The default value of `end` is set to the newline character `#!py \n`[^1]. So, whenever the end argument is not explicitly specified in the print function, a newline is appended to the input expression by default. In the code given above, by setting `end` to be a comma, we are forcing the `#!py print()` function to insert a comma instead of a newline at the end of the expression passed to it. It is called `end` because it is added at the end. To get a better picture, consider the following code:
```

Comment:
Hmm maybe yeah I thought of using annotations too, but then I feel that It's kinda missable - I mean it could go unnoticed right? You have to click on the plus to reveal the info.. I myself didn't notice those in the mkdocs documentation the first few times I saw it xD.

Reply:
I am trying to use the annotation feature here.. but it isn't rendering properly for me:

![image](https://github.com/bsc-iitm/python-textbook/assets/78845005/e1bc83c5-e19f-45be-8bc8-05a47408b891)

![image](https://github.com/bsc-iitm/python-textbook/assets/78845005/fc44c750-8832-410a-8d56-75e95fb52f41)

Reply:
I reinstalled all the packages.. works fine now. Let's use this 👍